word2vec_decade_experiment/02_word2vec_decade_analysis.py

```diff
@@ -16,58 +16,147 @@
 # # Analyze Word2Vec by Decades Run
 
 # +
+# %load_ext autoreload
+# %autoreload 2
+
 from pathlib import Path
+import pickle
+import itertools
 
+from gensim.models import Word2Vec
 import numpy as np
 import pandas as pd
+from scipy.spatial.distance import cdist, pdist, squareform
+from scipy.linalg import orthogonal_procrustes
+import tqdm
 import plotnine as p9
-from sklearn.metrics import (
-    auc,
-    roc_curve,
-    precision_recall_curve,
+
+from biovectors_modules.word2vec_analysis_helper import (
+    get_global_distance,
+    get_local_distance,
+    window,
 )
 # -
 
-# ## Load and Parse the performance data
+# ## Load Models and Parse Performance
 
-decade_df_dict = {}
-years = [1971, 1981, 1991, 2001, 2011]
+# Align word2vec Models since cutoff year
+year_cutoff = 2000
+aligned_model_file_path = "output/aligned_word_vectors.pkl"
+token_occurence_file = "output/earliest_token_occurence.tsv"
+year_distance_folder = "year_distances"
```

Comment:
This is another minor and personal style comment: I like to have nearly at the top of the notebook a section called "Settings", where I put the parameters that can be changed. Otherwise, they get lost in the middle of the code and it is hard to find them later.

Reply:
This isn't a bad idea; however, I'll default towards my settings at each individual section. Makes most sense in my head.